web/models/stable_diffusion/main.py

```diff
@@ -250,16 +248,15 @@ def stable_diff_inf(
         if DEBUG:
             log_write.write(f"time={step_ms}ms")
         latents = scheduler.step(noise_pred, i, latents)["prev_sample"]
-        # scale and decode the image latents with vae
-        latents = 1 / 0.18215 * latents
```

Comment:
For reference, looking at this shows why it was generating incorrect images with live preview. Because the latents are used as an input for the next step, doing this multiplication scaled up the values for the next input and likely overflowed the FP16 precision, as well as broke FP32 correctness :)

Reply:
yeah, got it. Thanks :)